### PR TITLE
feat: show current vote

### PR DESCRIPTION
**Pull request by Tuditi**
## Summary

Show the current vote of a user.

...

## Changelog

```
- Added setting current vote
- Changed communication between DetailsView<>ProposalAnswer<>ProposalQuestion
- Added UI changes wrt selected and voted for answer
```

### Platforms

> Please select any platforms where your changes have been tested.

- __Desktop__
	- [ ] MacOS
	- [x] Linux
	- [ ] Windows
- __Mobile__
	- [ ] iOS
	- [ ] Android

### Instructions

> Please describe the specific instructions, configurations, and/or test cases necessary to __test and verify__ that your changes work as intended.

...

## Checklist

> Please tick all of the following boxes that are relevant to your changes.

- [x] I have followed the contribution guidelines for this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added or modified tests that prove my changes work as intended
- [ ] I have verified that new and existing unit tests pass locally with my changes
- [x] I have verified that my latest changes pass CI workflows for testing and linting
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: packages/desktop/views/dashboard/governance/views/DetailsView.svelte [hunk 1]
```diff
@@ -55,27 +56,24 @@
...
-                }
-            )
-            totalVotes = votes?.reduce((accumulator, votes) => accumulator + votes, 0) ?? 0
-        } else {
-            totalVotes = 0
-        }
+        const votes = trackedParticipations.map(({ amount, startMilestoneIndex, endMilestoneIndex }) => {
+            const endMilestone = endMilestoneIndex <= 0 ? $networkStatus?.currentMilestone : endMilestoneIndex
+            return parseInt(amount) * (endMilestone - startMilestoneIndex)
+        })
```
Comment (maxwellmattryan) [inline packages/desktop/views/dashboard/governance/views/DetailsView.svelte:66]: Might be worth putting this into its own `utils` function in the `governance` module.

File: packages/shared/components/ProposalQuestion.svelte [hunk 2]
```diff
@@ -2,46 +2,48 @@
...
+    export let selectedAnswerValue: number = undefined
+    export let votedAnswerValue: number = undefined
 
     let percentages: string[]
-    let voteValue: number // TODO: get the current answer being voted by the account
 
     $: answers = [...question?.answers, { value: 0, text: 'Abstain', additionalInfo: '' }]
-    $: showMargin = isOpened || ((voteValue || voteValue === 0) && !isOpened) // voteValue 0 corresponds to abstained vote
-    $: currentVote, setPercentagesFromAccumulated()
+    $: showMargin = isOpened || ((votedAnswerValue || votedAnswerValue === 0) && !isOpened) // votedAnswerValue 0 corresponds to abstained vote
```
Comment (maxwellmattryan) [inline packages/shared/components/ProposalQuestion.svelte:19]: Might be worth making a constant e.g. `ABSTAIN_ANSWER_VALUE` so we can remove this comment and use it elsewhere as needed.

File: packages/shared/components/ProposalQuestion.svelte [hunk 3]
```diff
@@ -2,46 +2,48 @@
...
+    $: allVotes, setPercentagesFromAccumulated()
 
     function setPercentagesFromAccumulated(): void {
```
Comment (maxwellmattryan) [inline packages/shared/components/ProposalQuestion.svelte:22]: Accumulated what? Need just slightly more specificity in the function name / variables below

**Review by maxwellmattryan — CHANGES_REQUESTED** [review]
Looks pretty good, just left some minor comments.

**Comment by maxwellmattryan** [comment]
Some observations after testing:

- The current voted answer is shown, but the indicator icon is missing the pulse animation.
- It's a bit strange that I can select / de-select the current voted answer whenever the question is collapsed - in the collapsed state it shouldn't be possible to de-select the answer IMO. It looks slightly strange if I expand a question, select a different answer, collapse the question, resulting in the current voted answer looking different from the others.
- When clicking on a proposal to see its details, the loading of the questions itself takes a noticeably long amount of time plus another noticeably long amount of time to display the current votes.

Those are all relatively small details that will help improve UX, but as of now it is looking really good. Great job :clap:

**Comment by Tuditi** [comment]
I disabled the possibility to click on an answer when the question is disabled. I added tasks for the other remarks you made :relaxed: .

**Comment by maxwellmattryan** [comment]
Perhaps is out of the scope of this PR just slightly, but one last piece of feedback - if I want to re-vote, with a single changed answer, I have to manually re-select the answers for every question even if they're already selected. I should be able to just select a different answer for only one question and have the "Vote" button become enabled.

**Review by maxwellmattryan — APPROVED** [review]
LGTM :fire: 

I added remaining polish tasks in #5546

**Pull request merged**